homeassistant/components/media_player/bluesound.py

```diff
@@ -119,6 +154,44 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
                 hass, async_add_devices, host.get(CONF_HOST),
                 host.get(CONF_PORT), host.get(CONF_NAME, None))
 
+    @asyncio.coroutine
+    def async_service_handler(service):
+        """Map services to method of Bluesound devices."""
+        method = SERVICE_TO_METHOD.get(service.service)
+        if not method:
+            return
+
+        params = {key: value for key, value in service.data.items()
+                  if key != ATTR_ENTITY_ID}
+        entity_ids = service.data.get(ATTR_ENTITY_ID)
+        if entity_ids:
+            target_players = [player for player in hass.data[DATA_BLUESOUND]
+                              if player.entity_id in entity_ids]
+        else:
+            target_players = hass.data[DATA_BLUESOUND]
+
+        update_tasks = []
+        for player in target_players:
+            yield from getattr(player, method['method'])(**params)
+
+        for player in target_players:
+            if player.should_poll:
```

Comment:
Why check this? You know if bluesound entities should be polled or not.

Reply:
Very good question. I can't remember why, I will remove this